### PR TITLE
650 course statistic changes(connects #650)

### PR DESCRIPTION
**Pull request by Pr0chin**
- Changing layout regarding to Course Statistics Table

### CODE REVIEW

**Comment by Pr0chin** [comment]
capture

**Comment by Pr0chin** [comment]
capture

**Comment by Pr0chin** [comment]
- [x] Fixed some bugs related to data fetching
- [x] Changed table layout
- [x]  Implemented some major features in a Table
- [ ] Spreadsheet implementation

**Comment by Pr0chin** [comment]
capture

File: app/MyApp/app/views/CoursesStatistics.js [hunk 1]
```diff
@@ -9,37 +9,43 @@ $(function() {
...
         render: function() {
             var memberName=[]
             var memberStep= []
             var membersstatus=[]
             var stepName=[]
             var stepid = []
-            var failarr = []
+            var totalerrors = []
             var totalFailStatus = []
+            var you = []
```
Comment (lmmrssa) [inline app/MyApp/app/views/CoursesStatistics.js]: where is this variable used?

File: app/MyApp/app/views/CoursesStatistics.js [hunk 2]
```diff
@@ -48,6 +54,7 @@ $(function() {
                         async: false
                     });
                    memberName.push(members.toJSON().firstName + ' ' + members.toJSON().lastName)
+
```
Comment (lmmrssa) [inline app/MyApp/app/views/CoursesStatistics.js]: extra line

File: app/MyApp/index.html [hunk 3]
```diff
@@ -1208,12 +1210,22 @@
...
                 </tr>
             <% } %>
+             <tr>
+                <td><b>Total Errors</b></td>
+                <td></td>
+                <% for(var i = 0; i < stepName.length; i++) {%>
+                    <td style="color: red;"><b><%=TotalmemberSteperror[i]%></b></td>
+                <%}%>
+            </tr>
+
```
Comment (lmmrssa) [inline app/MyApp/index.html:1227]: extra lines

**Review by lmmrssa — APPROVED** [review]

**Pull request closed without merging**